english/cricket/cricket.md

```diff
@@ -1 +1,3 @@
 Click [here](http://www.espncricinfo.com) to catch up on cricket news and view live scores.
+
+You look around and find the cricket bat-and-ball game equipment and rule book. Click [here](../cricket-game/cricket-game.md) to read the rules.
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.